core-dev/packages/coq.8.2.dev/opam

```diff
@@ -2,6 +2,7 @@ opam-version: "1.1"
 maintainer: "hugo.herbelin@inria.fr"
```

Comment:
The `opam lint` command complains here, it wants 1.2 (as was done for other versions of the package)